### DISCUSSION TITLE
How can I make a sidebar wider?

### DISCUSSION THREAD

**Discussion by t1** [bigskysoftware/missing#37] [Q&A] [answered]
Missing is just cool. Thanks!

I need a little bit more space in my sidebar. I didn't find a hint in the docs and my experiments with using a grid actually flipped it to horizontal 🤔. Do I really have to resort to a width style attribute?

Whatever the solution, it might be worth adding a hint to the docs.

**Comment by dz4k** [comment, marked as answer]
The sidebar width is hardcoded at the moment -- as a workaround, you can add `grid-template-columns: <sidebar width> auto;` to the element that has `.sidebar-layout`.
